frontend/appflowy_flutter/lib/shared/icon_emoji_picker/flowy_icon_emoji_picker.dart

```diff
@@ -145,6 +147,7 @@ class _FlowyIconEmojiPickerState extends State<FlowyIconEmojiPicker>
 
   Widget _buildIconPicker() {
     return FlowyIconPicker(
+      requireIconBackgroundColor: widget.requireIconBackgroundColor,
       onSelectedIcon: (iconGroup, icon, color) {
         debugPrint('icon: ${icon.toJson()}, color: $color');
```

Comment:
please help me remove this `debugPrint`. :)

Reply:
Of course!